ArduCopter/Parameters.cpp

```diff
@@ -763,8 +763,8 @@ const AP_Param::GroupInfo ParametersG2::var_info[] = {
 
     // @Param: THROW_TYPE
     // @DisplayName: Type of Type
-    // @Description: Used by Throw mode. Specifies whether Copter is thrown upward or dropped.
-    // @Values: 0:Upward Throw,1:Drop
+    // @Description: Used by Throw mode. Specifies whether Copter is thrown upward, dropped or like a Frisbee.
```

Comment:
I think should be "thrown like a Frisbee"

Reply:
I'll make the change, my english is not perfect. 
What would be a correct commit message ?

Reply:
These are all in Copter, so should be something like

Copter: add a new "Frisbee" throw type to throw mode

Reply:
ok done and commits are squashed